lib/RoundedCornersEffect.vala

```diff
@@ -59,18 +66,74 @@ public class Gala.RoundedCornersEffect : Clutter.ShaderEffect {
             actor.notify["width"].connect (update_actor_size);
             actor.notify["height"].connect (update_actor_size);
 
+            update_clip_radius ();
             update_actor_size ();
         }
     }
 
+    private void update_clip_radius () requires (actor != null) {
+        var resource_scale = actor.get_resource_scale ();
+        set_uniform_value ("clip_radius", Utils.scale_to_int (clip_radius, monitor_scale / resource_scale));
```

Comment:
Why do we divide by the resource scale here?

Reply:
With https://github.com/elementary/gala/pull/2664, and I checked that removing monitor_scale from Utils.scale_to_int (clip_radius, monitor_scale ...) produces incorrect corner radius too

Reply:
Huh that's weird because I got this (which looks pretty right to me) with scale monitor framebuffer, 2x scale on this branch but with all of the get_resource_scale usage removed both in clip radius and actor size.
<img width="753" height="468" alt="Screenshot from 2025-11-25 20 39 59" src="https://github.com/user-attachments/assets/5737ebf1-9efe-465a-8f23-2763ba9796ec" />
whereas with the exact same settings on this branch but without any modifications i.e. with dividing by resource scale i get this 
<img width="1216" height="675" alt="Screenshot from 2025-11-25 20 43 55" src="https://github.com/user-attachments/assets/6e456bac-affc-45a5-a20d-5e492e5a4378" />